frontends/main/src/app-pages/DashboardPage/CoursewareDisplay/types.ts

```diff
@@ -0,0 +1,52 @@
+const DashboardResourceType = {
+  Course: "course",
+  Program: "program",
+} as const
+type DashboardResourceType =
+  (typeof DashboardResourceType)[keyof typeof DashboardResourceType]
+
+const EnrollmentStatus = {
+  NotEnrolled: "not_enrolled",
+  Enrolled: "enrolled",
+  Completed: "completed",
+} as const
+type EnrollmentStatus = (typeof EnrollmentStatus)[keyof typeof EnrollmentStatus]
+
+const EnrollmentMode = {
+  Audit: "audit",
+  Verified: "verified",
+} as const
+type EnrollmentMode = (typeof EnrollmentMode)[keyof typeof EnrollmentMode]
+
+type DashboardCourse = {
+  id: string
+  title: string
+  type: typeof DashboardResourceType.Course
+  run: {
```

Comment:
Maybe we want to have the ID of the run on here as well? Might be useful for making associations later on, or debugging.